en-US/firefox/features/fast.lang

```diff
@@ -18,18 +16,17 @@ Our new, powerful multi-process platform handles all your tabs without slowing d
 Firefox is now faster and leaner
 
 
+# Working out means exercising to get lighter and more proficent.  Here it means Firefox is lighter thus the speed faster.
 ;We’ve been working out, so you can get more done.
 We’ve been working out, so you can get more done.
 
 
-;Features
-Features
-
-
 ;Use less memory
 Use less memory
 
 
+# A hog is something that claims or takes all of everything.
```

Comment:
I would add: here it references to a program that takes up all the resources and slow down the computer.